9.mAP/notebooks/4.generate_map_scores_morphology_treatment.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<p>Consider removing the print statements that are commented out!</p><p> </p><p>Another thing that I am noticing is the total number of positive pairs. It's really small. Is this consistent throughout the analysis? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
there are only 4-8 well-level replicates for each treatment so yes. The cell death grouped treatments have a more robust sample size